PassengerJobs/Patches/JobSaveManagerPatch.cs

```diff
@@ -162,7 +162,7 @@ public static bool LoadJobChainPrefix(JobChainSaveData chainSaveData, List<JobBo
 
             PassengerHaulJobDefinition jobDefinition = jobChainHolder.AddComponent<PassengerHaulJobDefinition>();
             var chainData = new ExpressStationsChainData(jobData.originStationId, jobData.destinationStationIds);
-            jobDefinition.PopulateBaseJobDefinition(logicStation, jobData.timeLimitForJob, jobData.initialWage, chainData, LicenseInjector.License.v1);
+            jobDefinition.PopulateBaseJobDefinition(logicStation, jobData.timeLimitForJob, jobData.initialWage, chainData, LicenseInjector.License2.v1);
```

Comment:
I believe the `requiredLicense` could be either passengers 1 or 2 based on the `jobData.routeType`, the method name is misleading because it handles both express and rural route data